java/code/src/com/redhat/rhn/frontend/security/PxtAuthenticationService.java

```diff
@@ -73,6 +73,7 @@
         set.add("/rhn/common/DownloadFile");
         // search (safe to be unprotected, since it has no modifying side-effects)
         set.add("/rhn/Search.do");
+        set.add("/rhn/manager/upload/image");
```

Comment:
do you want to allow unprotrected download the image? don't we need up have authentication on this one?

Reply:
If I understand it correctly, it disables only CSRF token checking. The authentication is still enabled by `withImageAdmin` in the controller.

The HTTP API PR does the same:
https://github.com/uyuni-project/uyuni/pull/4910/files#diff-1d51cadaefa85c656ae781cea467ff6cd196d90d322108b18a15004d05a96252R76

Reply:
you are completely correct. sorry for the noise